core/docs/api/keyboards.md

```diff
@@ -102,28 +102,35 @@ typedef struct {
 
 -------------------------------------------------------------------------------
 
-# km_core_keyboard_load() {#km_core_keyboard_load}
+# km_core_keyboard_load_from_blob() {#km_core_keyboard_load_from_blob}
 
 ## Description
 
-Parse and load keyboard from the supplied path and a pointer to the loaded keyboard
-into the out paramter.
+Parse and load keyboard from the supplied blob and size and a pointer to the
+loaded keyboard into the out paramter.
```

Comment:
I'd omit "and size" since `blob` and it's size belong together. Or if the intention is to mention all parameters we'll have to mention the keyboard name as well. Overall I find this sentence hard to understand. What about something like

```suggestion
Parse and load a keyboard from the supplied blob and return a pointer to the
loaded keyboard in the out parameter.
```

Reply:
agreed, size isn't an optional parameter so is pretty clear